ppa/archive/import_util.py

```diff
@@ -178,10 +185,11 @@ class HathiImporter(DigitizedWorkImporter):
         }
     )
 
-    def __init__(self, source_ids=None, rsync_output=False):
+    def __init__(self, source_ids=None, rsync_output=False, output_dir=""):
         super().__init__(source_ids)
         # track whether (and how much) rsync output is desired
         self.rsync_output = rsync_output
+        self.output_dir = output_dir
```

Comment:
For the purposes of initialization, it seems like there should be an additional constraint that if `rsync_output` is set then `output_dir` must also be specified (and exist).

Reply:
previously it was putting the file in the current working directory - but you're right, better to be explicit